Sdk/Conversation/SwrveInputMultiValue.m

```diff
@@ -122,7 +128,28 @@ -(CGFloat) heightForRow:(NSUInteger) row {
         CGFloat h = (float)ceil(possibleSize.height);
         return h;
     } else {
-        return 51.0;
+        NSUInteger finalRow = row - ([self hasDescription]? 1 : 0);
+        NSDictionary *dict = [self.values objectAtIndex:finalRow];
+        NSString *cellText = [dict objectForKey:@"answer_text"];
+        UIFont *cellFont = [UIFont fontWithName:@"Helvetica" size:17.0];
+        CGSize constraintSize = CGSizeMake(280.0f, MAXFLOAT);
+        
+        CGFloat height = 51;
+        if ([SwrveInputMultiValue SwrveSystemVersionGreaterOrEqualThan:@"7.0"]) {
+            NSAttributedString *attributedText = [[NSAttributedString alloc] initWithString:cellText
+                                                                                 attributes:@{
+                                                                                              NSFontAttributeName: cellFont
+                                                                                              }];
+            CGRect rect = [attributedText boundingRectWithSize:CGSizeMake(tableView.bounds.size.width, CGFLOAT_MAX)
+                                                       options:NSStringDrawingUsesLineFragmentOrigin
+                                                       context:nil];
+            height = ceil(rect.size.height);
+        } else {
+            CGSize labelSize = [cellText sizeWithFont:cellFont constrainedToSize:constraintSize lineBreakMode:NSLineBreakByWordWrapping];
+            height = ceil(labelSize.height);
+        }
+        
+        return height + 22;
     }
```

Comment:
Wop! Maybe sizeWithFont constrainedToSize works everywhere (even in iOS6 where the other method is deprecated)